src/panels/config/backup/ha-config-backup-location.ts

```diff
@@ -43,13 +44,13 @@ class HaConfigBackupDetails extends LitElement {
 
   @state() private _error?: string;
 
-  protected firstUpdated(changedProps) {
-    super.firstUpdated(changedProps);
-
-    if (this.agentId) {
-      this._fetchAgent();
-    } else {
-      this._error = "Agent id not defined";
+  protected updated(changedProps: PropertyValues): void {
```

Comment:
Instead of `updated` you can also use `willUpdate`, that will speed it up a bit. In general, if you dont need the DOM, use `willUpdate`.

Reply:
It will not change a lot as it's asynchronous. I will make the change 🙂